src/mmw/js/src/analyze/views.js

```diff
@@ -30,6 +33,50 @@ var ResultsView = Marionette.LayoutView.extend({
         analyzeRegion: '#analyze-tab-contents'
     },
 
+    ui: {
+        'modelPackageLinks': 'a.model-package',
+    },
+
+    events: {
+        'click @ui.modelPackageLinks': 'selectModelPackage',
+    },
+
+    selectModelPackage: function (e) {
+        e.preventDefault();
+
+        var modelPackages = settings.get('model_packages'),
+            modelPackageName = $(e.target).data('id'),
+            modelPackage = _.find(modelPackages, {name: modelPackageName}),
+            newProjectUrl = '/project/new/' + modelPackageName,
+            projectUrl = '/project';
+
+        if (!modelPackage.disabled) {
+            if (settings.get('itsi_embed') && App.currentProject && !App.currentProject.get('needs_reset')) {
+                var currModelPackageName = App.currentProject && App.currentProject.get('model_package');
```

Comment:
Won't this set `currModelPackageName` to `true` if succesful, instead of the actual name?